Changing the load screen slightly to not have a version number

evertvorster opened the pull request:
A very minor edit, just removed the version number and incorrect URL from the game load screen

BenjamenMeyer commented:
As it's a binary file change one of is will need to checkout the branch to run and confirm.

nabaco commented:
@evertvorster to speed up the review, could you add a screenshot of before and after?

evertvorster commented:
Sure thing. menu.png is the actual original file. It's just a png, renamed to a .image extension. Attached below for your perusal. 
![menu](https://user-images.githubusercontent.com/12045269/86889429-01a36b00-c0fc-11ea-93b9-4bafbe03fb55.png)

The file I provided is in .dds format, with BC1 compression. Tested with the gaming engine, and it works. 
I converted it to .jpg now for purposes of illustration, and because it is a MUCH smaller upload. It is now called menu.jpg.
![menu](https://user-images.githubusercontent.com/12045269/86889721-80000d00-c0fc-11ea-95b6-573e04a98d7f.jpg)
